.circleci/config.yml

```diff
@@ -49,7 +31,7 @@ commands:
           name: "Push new app in dark mode"
           command: |
             export DOCKER_IMAGE=tariff-backend
-            export DOCKER_TAG=<<# parameters.dev-release >>dev-<</ parameters.dev-release >>${CIRCLE_SHA1}
+            export DOCKER_TAG="<< parameters.docker_image_tag >>"
```

Comment:
Had this thought when I was doing search query parser. Why not just explicitly pass it in everywhere :tada: . The old syntax is awful too

Reply:
Yeh, probably a good idea - I'd rather limit the scope of changes in this PR though so may look to do that in a follow up PR